root/doc/DocError.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+import bugTrackerURL from '../static/scripts/common/utility/bugTrackerURL';
+
+import DocSearchBox from './components/DocSearchBox';
+
+type Props = {|
+  +$c: CatalystContextT,
+  +id: string,
+  +useGoogleCustomSearch: boolean,
+|};
+
+const DocError = ({
+  $c,
+  id,
+  useGoogleCustomSearch,
+}: Props) => {
+  return (
+    <Layout fullWidth title={l('Page Not Found')}>
+      <div className="wikicontent" id="content">
+        {useGoogleCustomSearch ? <DocSearchBox /> : null}
+
+        <h1>{l('Page Not Found')}</h1>
+
+        <p>
+          <strong>
+            {texp.l('Sorry, “{id}” is not a valid documentation page.',
+                    {id: id.replace(/_/g, ' ')})}
```

Comment:
Need to use a regex because for some reason Javascript will otherwise only replace the first appearance (which seems stupid, but eh).